CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Fixed
+
+- Videos uploaded in other formats than 16/9 were distorted to fit that
+  ratio. They now just have black bars.
```

Comment:
Need to update now that we changed strategy?

Reply:
You mean running a migration ?

Reply:
No I meant that now format with height more than 9:16 of width will adapt and formats with height less than 9:16 of width will have vertical black bars.